tests/e2e/core-tests/product_simple_02_product_functionality_test.js

```diff
@@ -5,8 +5,8 @@ Feature('Simple product fucntionality');
 Before(({ I }) => { // or Background
     I.loginAsVendor();
 });
-Scenario('simple product functional', ({ I }) => {
-
+Scenario('simple product functional', ({ I,loginAs}) => {
+    loginAs('vendor');
```

Comment:
Maintain upper and lower case for the user role because the user is case sensitive.